docs/changelog/116712.yaml

```diff
@@ -0,0 +1,11 @@
+pr: 116712
+summary: Remove apm_user role
+area: APM
+type: breaking
+issues:
+  - 116712
+breaking:
+  title: Remove apm_user role
+  area: APM
+  details: The apm_user role have been removed. Instead stack-wide roles (viewer, editor and superuser) should be used.
+  impact: Any user using the apm_user role should replace it by one of the stack-wide roles (viewer, editor and superuser) in order to have access to APM ui and APM data.
```

Comment:
```suggestion
  impact: Users depending on built-in `apm_user` role will lose access to APM UI and data.  In order to retain access, replace it by one of the existing built-in roles ({ref}/built-in-roles.html#built-in-roles-viewer[viewer], {ref}/built-in-roles.html#built-in-roles-editor[editor] or ref}/built-in-roles.html#built-in-roles-superuser[superuser]).
  notable: false
```

Reply:
@brianseeders are you converting asciidoc to md syntax for these entries for the 9.0+ release notes?

I guess we should start trying to get folks to write changelog entries in markdown going forward...

Reply:
I missed the fact that we moved away from asciidoc. Thanks for pointing that out.